pkg/types/leaderelection.go

```diff
@@ -48,7 +48,12 @@ type MemberLeaseCallbacks struct {
 }
 
 // IsLeaderCallbackFunc is type declaration for callback function to Check LeadershipStatus.
-type IsLeaderCallbackFunc func(context.Context, *EtcdConnectionConfig, time.Duration, *logrus.Entry) (bool, error)
+type IsLeaderCallbackFunc func(context.Context, *EtcdConnectionConfig, time.Duration, *logrus.Entry) (bool, bool, error)
```

Comment:
Rename this, since it now also checks for learner status, not just leadership status.